### ISSUE TITLE
MD029 always triggers in blockquote even if ordered list marking is correct

### ISSUE THREAD

**Issue by skyegecko** [markdownlint/markdownlint#150]
### Sample:

``` markdown
# MDL test quoted ordered list

> * UL 1
> * UL 2
>
> 1. OL 1
> 1. OL 2
```
### Result:

```
$ mdl test.md
test.md:6: MD029 Ordered list item prefix
test.md:7: MD029 Ordered list item prefix
```

This occurs both using default ruleset and my own ruleset, which is as follows:

``` ruby
# Style rule for my personal markdown style preferences for mdl
# https://github.com/mivok/markdownlint/blob/master/docs/creating_styles.md

# Start by including all rules
all

# Headers use ATX style only
rule 'MD003', :style => :atx

# Use 4 spaces for list indentation
rule 'MD007', :indent => 4

# Don't pay attention to line length
exclude_rule 'MD013'

# Permit inline HTML
exclude_rule 'MD033'
```

Test performed using mdl version 0.4.0 installed from rubygems.

**Comment by dsbert** [comment]
This also happens in nested lists

``` markdown
* Success
  1. pass
  2. pass
  3. pass
  4. pass
```

**Comment by markymc** [comment]
I've just experienced the same. Is there no workaround for this?

**Comment by jaymzh** [comment]
@dsbert - your example is not a bug, that's actually correct. It's because you didn't make all your ordered-lists the same. That should be:

```
* Success
  1. pass
  1. pass
  1. pass
  1. pass
```

**Comment by jaymzh** [comment]
The original-poster's bug is a valid one, and probably needs the same fix as for https://github.com/markdownlint/markdownlint/issues/117

**Comment by jaymzh** [comment]
This was fixed by #543
